Error: Cannot find module 'node:fs/promises'

## **BEFORE YOU SUBMIT** please read the following:
<!--
If you have a support request or question please submit them to
[StackOverflow](http://stackoverflow.com/questions/tagged/babeljs) using the tag
`[babel]` or
the [Babel Slack](babeljs.slack.com). Future support requests will be closed.
-->
*Delete what isn't applicable below*
**I'm submitting a bug report**


**Webpack Version:**
5.74.0

**Babel Core Version**:
7.19.6

**Babel Loader Version**:
9.0.0


**Please tell us about your environment:**
Windows 11

**Current behavior:**
Receiving the following error with babel-loader 9.0.0

```
ERROR in ./src/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'node:fs/promises
```

No issues observed with babel-loader 8.2.5

**Expected/desired behavior:**

There should not be any build error.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -23,6 +23,9 @@ jobs:
         node-version: [14, 16, 18]
         webpack-version: ['5']
         include:
+          - node-version: "14.15.0" # The minimum supported node version
+            webpack-version: latest
+            os: ubuntu-latest
           - node-version: latest
             webpack-version: latest
             os: ubuntu-latest

--- a/src/cache.js
+++ b/src/cache.js
@@ -13,7 +13,7 @@ const zlib = require("zlib");
 const crypto = require("crypto");
 const findCacheDir = require("find-cache-dir");
 const { promisify } = require("util");
-const { readFile, writeFile, mkdir } = require("node:fs/promises");
+const { readFile, writeFile, mkdir } = require("fs/promises");
 
 const transform = require("./transform");
 // Lazily instantiated when needed

